src/app/home/home.component.ts

```diff
@@ -38,16 +48,17 @@ export class HomeComponent implements OnInit {
         href: new URL('/anchor-match-fund/', environment.blogUriPrefix)
       }
     },
-    {
-      headerText: 'One donation. Twice the impact.',
-      bodyText: "You donate.\nWe double it.",
-      iconColor: "primary",
-      backgroundImageUrl: new URL('/assets/images/blue-texture.jpg', environment.donateGlobalUriPrefix),
-      button: {
-        text: "Explore now",
-        href: new URL('/explore', environment.donateGlobalUriPrefix),
-      }
-    },
+    // TODO: hould replace Anchor Mach Fund card on 2nd October 2023
+    // {
+    //   headerText: ' Applications for Arts for Impact are now open!',
+    //   bodyText: "Apply by 3rd November 2023",
+    //   iconColor: "primary",
+    //   backgroundImageUrl: new URL('/assets/images/red-coral-texture.png', environment.donateGlobalUriPrefix),
+    //   button: {
+    //     text: "Apply now",
+    //     href: new URL('/artsforimpact', environment.donateGlobalUriPrefix),
```

Comment:
This needs to be the blog uri not the donate uri
```suggestion
    //     href: new URL('/artsforimpact', environment.environment.blogUriPrefix),
```

And should it be commented out or do we actually want to publish this link now?